assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -171,6 +171,7 @@ export const plotSeriesSelectTable = function(elem,
             tr.append('td')
                 .append('a')
                     .attr('href', param => `${config.WATERALERT_SUBSCRIPTION}/?site_no=${siteno}&parm=${param.parameterCode}`)
+                    .attr('class', 'usa-tooltip usa-link')
```

Comment:
This should not have usa-tooltip as a style. It's not a tooltip.

Reply:
Confused about this one. I thought I was following the guidance here: https://designsystem.digital.gov/components/tooltip/ .

Reply:
Is it supposed to have a tooltip? If it is, you are rendering dynamically and will need to initialize the tooltip (see how it is done in d3render/tooltips.js. I'm going to update that one because you really should pass the element to which you are adding the tooltip. So here it would be components.tooltip.init(tr.node()).

Reply:
Oooh, that would be why the tooltip didn't have the behavior I was expecting -- I thought it was just a browser display difference. And yes, the product owners, added that they wanted a tooltip added, after you did your first review.